assets/airports/kabq.json

```diff
@@ -0,0 +1,2170 @@
+    },
+    "runways":[
+        {
+            "name": ["3", "21"],
+            "end": [
+                ["N35d01m20.09", "W106d37m50.17", "5305ft"],
+                ["N35d02m30.27", "W106d36m25.46", "5316ft"]
+            ],
+            "ils": [true, true]
+        },
+        {
+            "name": ["8", "26"],
+            "end": [
+                ["N35d02m39.67", "W106d37m17.72", "5312ft"],
+                ["N35d02m38.63", "W106d34m31.89", "5355ft"]
+            ],
+            "ils": [true, true]
+        },
+        {
+            "name": ["12", "30"],
+            "end": [
+                ["N35d02m36.72", "W106d37m14.71", "5312ft"],
+                ["N35d01m59.5", "W106d36m18.53", "5314ft"]
+            ],
+            "ils": [false, false]
+        },
+        {
+            "name": ["17", "35"],
+            "end": [
+                ["N35d03m27.83", "W106d36m39.6", "5312ft"],
+                ["N35d01m49.12", "W106d36m46.68", "5314ft"]
+            ],
+            "ils": [false, false]
+        }
+    ],
+    "sids": {
+        "ADYOS3": {
+            "icao": "ADYOS3",
+            "name": "Adios Three",
+            "rwy": {
+                "21": [["ASTAH", "A90-"], "LVSTA"],
+                "26": ["ABQ"]
+            },
+            "body": ["ADYOS"],
+            "exitPoints": {
+                "GUP": ["BABYY", "BUSLE", "GUP"],
+                "ZUN": ["PNYON", "ZUN"]
+            },
+            "draw": [
+                ["ASTAH", "LVSTA", "ADYOS", "BABYY", "BUSLE", "GUP*"],
+                ["ABQ", "ADYOS", "PNYON", "ZUN*"]
+            ]
+        },
+        "ABQ2": {
+            "icao": "ABQ2",
+            "name": "Albuquerque Two",
+            "rwy": {
+                "3": [],
+                "8": [],
+                "21": [],
+                "26": []
+            },
+            "body": [],
+            "exitPoints": {
+                "ALS": ["ALS"],
+                "CNX": ["CNX"],
+                "DRO": ["DRO"],
+                "FTI": ["FTI"],
+                "GUP": ["GUP"],
+                "ONM": ["ONM"],
+                "RSK": ["RSK"],
+                "SJN": ["SJN"],
+                "TAS": ["TAS"],
+                "TCC": ["TCC"],
+                "TXO": ["TXO"],
+                "ZUN": ["ZUN"]
+            },
+            "draw": [
+                ["ALS*"],
+                ["CNX*"],
+                ["DRO*"],
+                ["FTI*"],
+                ["GUP*"],
+                ["ONM*"],
+                ["RSK*"],
+                ["SJN*"],
+                ["TAS*"],
+                ["TCC*"],
+                ["TXO*"],
+                ["ZUN*"]
+            ]
+        },
+        "ATOMK3": {
+            "icao": "ATOMK3",
+            "name": "Atomkay Three",
+            "rwy": {
+                "21": [],
+                "26": []
+            },
+            "body": [["FOXRR", "A80+"], ["TYWEB", "A100+|S250-"], ["TKILA", "A115+|S250-"], "FIREY"],
```

Comment:
Regarding this restriction and the other ones in the SID's: They're there to make sure that aircraft don't speed up above 250 at TYWEB and then slow down for the actual restriction at TKILA. The restriction at TKILA also applies to all the fixes before it but last time I checked aircraft ignored that (Maybe it works now with all the changes made to VNAV?). Hence the restriction at TYWEB